admin_guide/topics/deploying-cluster-auto-scaler.adoc

```diff
@@ -0,0 +1,278 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/cluster-autoscaler.adoc
+
+[id='deploying-cluster-auto-scaler-{context}']
+= Deploying the auto-scaler components on your cluster
+
+After you create the Launch Configuration and Auto Scaling group, you can deploy
+the auto-scaler components onto the cluster.
+
+.Prerequisites
+
+* Install a {product-title} cluster in AWS.
+* Create a primed image.
+* Create a Launch Configuration and Auto Scaling group that reference the primed
+image.
+
+.Procedure
+To deploy the auto-scaler:
+
+. Update your cluster to run the auto-scaler:
+.. Add the following parameter to the inventory file that you used to create the
+cluster, by default *_/etc/ansible/hosts_*:
++
+----
+openshift_master_bootstrap_auto_approve=true
+----
+
+.. To obtain the auto-scaler components, run the playbook again:
++
+----
+$ ansible-playbook -i </path/to/inventory/file> \
+    /usr/share/ansible/openshift-ansible/playbooks/deploy_cluster.yml
+----
+
+.. Confirm that the `bootstrap-autoapprover` pod is running:
++
+----
+$ oc get pods --all-namespaces | grep bootstrap-autoapprover
+openshift-infra         bootstrap-autoapprover-0                         1/1       Running   0  
+----
```

Comment:
@kalexand-rh 
Should this output have headings, as in:
`NAME                                READY             STATUS      RESTARTS          AGE`

Reply:
Here in case you want them: 

```
NAMESPACE               NAME                                             READY     STATUS    RESTARTS   AGE
```